XmlFormatter/src/Settings/Adapter/PropertyAdapter.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using XmlFormatter.src.Interfaces.Settings.DataStructure;
+using XmlFormatter.src.Settings.DataStructure;
+
+namespace XmlFormatter.src.Settings.Adapter
+{
+    /// <summary>
+    /// This class is a adapter between the new setting methods and the old resource based
+    /// </summary>
+    class PropertyAdapter : ISettingScope
+    {
+        /// <summary>
+        /// The name of this scope
+        /// </summary>
+        private string name;
+
+        /// <summary>
+        /// Readonly access to the scope name
+        /// </summary>
+        public string Name => name;
+
+        /// <summary>
+        /// Create a new instance of this adapter which will always use "Default" as scope name
+        /// </summary>
+        public PropertyAdapter()
+        {
+            SetName("Default");
+        }
+
+        /// <summary>
+        /// Set the name of this adapter
+        /// </summary>
+        /// <param name="name">The new name to use</param>
+        public void SetName(string name)
+        {
+            this.name = name;
+        }
+
+        /// <summary>
+        /// Add a new setting to the resource file.
+        /// This will only work if the setting is already defined
+        /// </summary>
+        /// <param name="setting">The setting to add</param>
+        public void AddSetting(ISettingPair setting)
+        {
+            try
+            {
+                Type type = Properties.Settings.Default[setting.Name].GetType();
+                if (type == setting.Type)
+                {
+                    Properties.Settings.Default[setting.Name] = setting.Value;
+                }
+            }
+            catch (Exception)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5052706)